maintainers/maintainer-list.nix

```diff
@@ -6730,6 +6730,12 @@
     githubId = 3967312;
     name = "Jocelyn Thode";
   };
+  joedevivo = {
+    email = "55951+joedevivo@users.noreply.github.com";
+    github = "joedevivo";
+    githubId = 55951;
+    name = "Joe DeVivo";
+  };
```

Comment:
This is buried deep within the manual (and we should maybe add it to CONTRIBUTING.md too...), but additions to `maintainers/maintainter-list.nix` are supposed to be in their own commits titled "maintainers: add [name]": https://nixos.org/manual/nixpkgs/unstable/#var-meta-maintainers